crafters/image/ImageNormalizer/manifest.yml

```diff
@@ -2,12 +2,11 @@ manifest_version: 1
 name: ImageNormalizer
 kind: crafter
 type: pod
-description: it receives values of file names on the doc-level and returns image matrix
-  on the chunk-level
+description: Receives values of file names on the doc-level and returns image matrix on the subdocument-level
```

Comment:
same here no level involved

Reply:
The original docs talked about chunks, but is also not the case right? @JoanFM

Reply:
yes, no chunks involved

Reply:
Done